include/packetgraph/vtep.h

```diff
@@ -130,6 +134,13 @@ inline int pg_vtep_add_vni(struct pg_brick *brick,
 int pg_vtep_add_mac(struct pg_brick *brick, uint32_t vni,
 		    struct ether_addr *mac, struct pg_error **errp);
 
+/**
+ * Like what the function name say
+ * Arguments are are the same as pg_vtep_add_mac
+ */
+int pg_vtep_unset_mac(struct pg_brick *brick, uint32_t vni,
+		      struct ether_addr *mac, struct pg_error **errp);
```

Comment:
Add somes descriptions for doxygen documentation
@param description
@param description